rnd/autogpt_server/autogpt_server/executor/manager.py

```diff
@@ -47,6 +49,8 @@ def execute_node(loop: asyncio.AbstractEventLoop, data: Execution) -> ExecutionS
     exec_data = data.data
     node_id = data.node_id
 
+    agent_server: AgentServer = get_service_client(AgentServer)  # type: ignore
```

Comment:
get_service_client is an I/O (can be slow) so it's better to simply expose a shared field or a @property method like:
```
@property
def agent_server_client(self):
    return get_service_client(AgentServer)
```